### PR TITLE
Add manual testing, change tests

### PR DESCRIPTION
**Pull request by huyuxin0429**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -902,10 +902,41 @@ testers are expected to do more *exploratory* testing.
...
-   1. Prerequisite: Create an appointment e.g. `apmt add i/1 d/2022-12-31 1200`
-   2. Test case: Close and reopen the application <br>
-      Expected: Appointments reference the same patients as previous session before it was closed.
+   1. If JSON files are missing, `Doc'it` will start with a sample AddressBook and AppointmentBook with sample Patients and sample Appointments respectively.
+       2. If JSON files are corrupted, `Doc;it` will start with a blank address book and blank appointment book.
+   2. Save appointment
+       1. Prerequisite: Create an appointment e.g. `apmt add i/1 d/2022-12-31 1200`
+       2. Test case: Close and reopen the application <br>
+          Expected: Appointments reference the same patients as previous session before it was closed.
+2. _{ more test cases … }_
```
Comment (gycgabriel) [inline docs/DeveloperGuide.md:911]:
```suggestion
```

To delete

**Review by gycgabriel — CHANGES_REQUESTED** [review]
Remove ```more test cases  ``` in multiple places

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (huyuxin0429) [reply docs/DeveloperGuide.md]: Removed

**Review by gycgabriel — APPROVED** [review]
lgtm

**Pull request merged**